dsaldonid/JobTracker
# Initial frontend setup

Notes

- Create the initial UI via `yarn create react-app job-tracker-frontend`
- Add the following libraries so we have most (if not all) the components we need: @mui/material, @mui/icons-material, @emotion/styled, @emotion/react, recharts.

QA:

- Build yarn, then run yarn start to verify the below web page appears:

![image](https://user-images.githubusercontent.com/11845812/213895793-2499d4fe-d712-423a-8172-45ffd705b4e0.png)
